config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -518,7 +518,7 @@ periodics:
   extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: release-1.7
+      base_ref: release-1.8
```

Comment:
In-progress test after this change:

- https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/cloud-provider-azure-ccm-windows-capz/1634222734634389504

Reminder to check back for results!